frontend/src/components/c-ramp.vue

```diff
@@ -267,4 +298,17 @@ export default defineComponent({
     }
   }
 }
+
+.ramp-padding {
+  height: 100%;
+  position: relative;
+}
+
+.date-indicators {
+  @include mono-font;
```

Comment:
Is there any reason we are using a monospaced font for the dates? I think we should follow the rest of the site, and use the default sans serif font for all dates.